integron-finder/1.5.1/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:16.04
+
+#####EXTRA LABELS#####
+LABEL autogen="no" \ 
+	software="Integron Finder" \ 
+    version="1" \
+	software.version="1.5.1" \ 
+	about.summary="Integron Finder aims at detecting integrons in DNA sequences." \ 
+	base_image="ubuntu:16.04" \
+    about.home="https://github.com/gem-pasteur/Integron_Finder" \
+    about.license="SPDX:GPL-3.0-only" \
+    extra.identifier.biotools="integron_finder" \ 
+    ANNOT.Name="Integron_Finder" \
+    ANNOT.Version="1.5.1" \
```

Comment:
could do some cleanup to avoid duplication in labels

Reply:
I was wondering if we wanted to keep those "ANNOT..." labels or if we don't care about them. If we don't care I'll remove them here (and in a couple other tools where they went unnoticed ;) )

Reply:
those ANNOT were bioshadocker related. EDMA ones could be kept, for record, but name, version, ... are duplicates with our LABEL